test/e2e/bench/bench.test.ts

```
@@ -56,47 +57,107 @@
     let subscriberSummaryEntry: any = null; // Capture testFinished entry
 
+    console.log(`[TEST] Test channel: ${testChannel}`);
+    console.log(`[TEST] API key exists: ${!!apiKey}`);
+    
     try {
       // 1. Start Subscriber (Restored original command)
+      console.log("[TEST] Creating subscriber promise");
       const subscriberPromise = new Promise<void>((resolveSubscriber, rejectSubscriber) => {
         // console.log("Spawning original subscriber command..."); // Debug
+        const subEnv = { ...process.env };
+        delete subEnv.ABLY_CLI_TEST_MODE;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)